src/src/BioSounds/Controller/Administration/RecordingController.php

```diff
@@ -104,6 +109,13 @@ public function save()
             throw new ForbiddenException();
         }
 
+        if (!preg_match('/^[0-9]\d*$/', $_POST['recording_gain_number'])) {
+            return json_encode([
+                'isValid' => 1,
+                'message' => 'Recording gain must be a positive integer.',
+            ]);
+        }
+
```

Comment:
Here the code where the recording gain has to be set as positiv integer!

Reply:
OK I corrected this, thanks